.github/workflows/php.yml

```diff
@@ -42,25 +44,12 @@ jobs:
       with:
         php-version: ${{ matrix.php }}
 
-    - name: Validate composer.json and composer.lock
-      run: composer validate
-      
-    - name: Cache Composer packages
-      id: composer-cache
-      uses: actions/cache@v3
-      with:
-        path: vendor
-        key: ${{ runner.os }}-php${{ matrix.php }}-${{ hashFiles('**/composer.lock') }}
```

Comment:
Also, now that composer.lock is not available at this point, we need some other way of determining the version of the cache...

Reply:
used [this](https://github.com/shivammathur/setup-php?tab=readme-ov-file#cache-composer-dependencies) as reference, combining php version, composer.json to act as the [key](https://github.com/univapay/univapay-php-sdk/pull/67/commits/bde7cb11d9afbae444af321c18a2b8722558129c#diff-a73bb6555480a5ee79ae276a3f5d71a08fa316e09a4a8da7b643cf1e92c97df9R41-R42)